src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -436,6 +437,16 @@ private async Task StopAsyncCore(bool disposing)
             }
         }
 
+#if NETCOREAPP3_0
+        public async IAsyncEnumerable<object> StreamAsyncCore(string methodName, Type returnType, object[] args, CancellationToken cancellationToken = default)
+        {
+            var stream = (await StreamAsChannelCoreAsyncCore(methodName, returnType, args, cancellationToken)).ReadAllAsync();
```

Comment:
Call the public API `StreamAsChannelCoreAsync`, that way you get the `ForceAsync` and the log scope for free